lint/settings.py

```diff
@@ -81,25 +81,29 @@ def get_settings_objects():
     for name in sublime.find_resources("SublimeLinter.sublime-settings"):
         try:
             yield name, util.load_json(name, from_sl_dir=False)
-        except IOError as ie:
+        except IOError:
             util.printf("Settings file not found: {}".format(name))
-        except ValueError as ve:
+        except ValueError:
             util.printf("Settings file corrupt: {}".format(name))
 
 
 def validate_settings():
-    status_msg = "SublimeLinter - Settings invalid. Details in console."
+    status_msg = "SublimeLinter - Settings invalid!"
     schema_file = "resources/settings-schema.json"
     schema = util.load_json(schema_file, from_sl_dir=True)
-
+    window = sublime.active_window()
+    util.clear_message()
```

Comment:
Is this necessary? We don't do this for the other errors, and it would be tricky to decide when a previous error has been resolved.

Reply:
What this accomplishes, is that the message disappears when you're settings are valid again. This is the only case right now where we accurately detect that a problem has disappeared. I think it's important to provide this positive feedback in this particular case.

Reply:
👍